src/tests/test-profile/meson.build

```diff
@@ -0,0 +1,8 @@
+if build_standalone
+  install_data([
+      'thinkpad-p1-no-iommu.json.gz',
+      'thinkpad-p1-iommu.json.gz',
```

Comment:
whichever tree they end up would much rather store in tree uncompressed so that we can treat them as text and see differences if test data needs to be refreshed from new plugins and such.

Even if they're installed into a package, they can be compressed at build time instead.

Reply:
Yes, that's a good idea. We also need that in the "rename an ID" case too.